Example/AppDelegate.swift

```diff
@@ -31,10 +31,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
```

Comment:
Remove this function at all?

Reply:
Sorry, this was removed occasionally. We still need it for iOS < 11.